package.json

```diff
@@ -18,9 +18,8 @@
     "prepublish": "npm run build",
     "prepublish-watch": "npm run watch",
     "start": "./node_modules/.bin/webpack-dev-server",
-    "tap-integration": "./node_modules/.bin/tap ./test/integration/*.js",
-    "tap-unit": "./node_modules/.bin/tap ./test/unit/*.js",
-    "test": "npm run lint && npm run tap-unit && npm run tap-integration",
+    "tap": "./node_modules/.bin/tap ./test/{unit,integration}/*.js",
```

Comment:
Unfortunately coveralls requires all tests to be run at once in order to be able to generate a coverage report. This should resolve the issue where coveralls is reporting our coverage rate as far too low.